network/network_test.go

```diff
@@ -694,9 +760,24 @@ func TestNetwork_validateNodeDID(t *testing.T) {
 		cxt.network.nodeDID = *nodeDID
 		cxt.docResolver.EXPECT().Resolve(*nodeDID, nil).MinTimes(1).Return(completeDocument, &vdrTypes.DocumentMetadata{}, nil)
 
-		err := cxt.network.validateNodeDID(ctx, *nodeDID)
+		health := cxt.network.checkNodeDIDHealth(ctx, *nodeDID)
+
+		assert.Equal(t, core.HealthStatusDown, health.Status)
+		assert.Equal(t, "none of the DNS names in TLS certificate match the NutsComm service endpoint (nodeDID=did:nuts:test, NutsComm=grpc://not.nuts.nl:5555)", health.Details)
+	})
+	t.Run("error - NutsComm unavailable", func(t *testing.T) {
+		ctrl := gomock.NewController(t)
+		cxt := createNetwork(t, ctrl)
+		_ = cxt.keyStorage.SavePrivateKey(ctx, keyID.String(), certificate.PrivateKey)
+		cxt.network.certificate = certificate
+		cxt.network.nodeDID = *nodeDID
+		cxt.docResolver.EXPECT().Resolve(*nodeDID, nil).MinTimes(1).Return(completeDocument, &vdrTypes.DocumentMetadata{}, nil)
+		cxt.network.selfTestDialer.NetDialer.Timeout = 10 * time.Millisecond
+
+		health := cxt.network.checkNodeDIDHealth(ctx, *nodeDID)
 
-		assert.EqualError(t, err, "none of the DNS names in TLS certificate match the NutsComm service endpoint (nodeDID=did:nuts:test, NutsComm=grpc://not.nuts.nl:5555)")
+		assert.Equal(t, core.HealthStatusUnknown, health.Status)
+		assert.Contains(t, health.Details, "cannot connect to own NutsComm grpc://nuts.nl:5555: dial tcp")
```

Comment:
this will try to connect to `grpc://nuts.nl:5555` every time the test runs, not sure if this matters